### PR TITLE
Delete sites references

### PR DESCRIPTION
**Pull request by stephamon**
Part of #1910

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-df13cac340/how-to/)

File: content/cloud-databases/cloud-databases-faq.md [hunk 1]
```diff
@@ -13,12 +13,10 @@ product_url: cloud-databases
```
Comment (nimbinatus) [inline content/cloud-databases/cloud-databases-faq.md]: Should the mod date be updated?

File: content/cloud-databases/public-and-private-access-for-cloud-databases.md [hunk 2]
```diff
@@ -20,8 +20,7 @@ Cloud Database using any of the following methods:
 -   **Connect using the database Hostname string (private access within
```
Comment (nimbinatus) [inline content/cloud-databases/public-and-private-access-for-cloud-databases.md]: Should the mod date be updated?

File: content/cloud-dns/cloud-dns-faq.md [hunk 3]
```diff
@@ -39,10 +39,6 @@ Cloud DNS is currently available at no additional cost, and is intended for use
```
Comment (nimbinatus) [inline content/cloud-dns/cloud-dns-faq.md]: Should the mod date be updated?

File: content/cloud-dns/managed-dns-faq.md [hunk 4]
```diff
@@ -65,10 +65,6 @@ For example, it is possible to configure a Managed DNS (Early Access) zone with
```
Comment (nimbinatus) [inline content/cloud-dns/managed-dns-faq.md]: Same question on down...

File: content/cloud-load-balancers/using-the-rackspace-network-effectively.md [hunk 5]
```diff
@@ -38,8 +36,7 @@ offer the first level of network security. For example if you're not
 expecting external traffic to your Cloud Servers, you can create a
 firewall around them and reduce their exposure to outside threats. Note
 that Rackspace creates all new servers with basic security settings,
-however you should follow the instructions located in [Configuring Basic
-Security](/how-to/configuring-basic-security-0)to
+however you should follow the instructions located in [Configuring Basic Security](/how-to/configuring-basic-security)to
```
Comment (nimbinatus) [inline content/cloud-load-balancers/using-the-rackspace-network-effectively.md]: space needed between the ending `)` and the word `to`

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-cb9a1047a8/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-71e05108ac/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-71e05108ac/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-efbe960b2b/how-to/)

**Pull request merged**